mec-17.4.1-recommendation-systems-mini-project/Mini_Project_Recommendation_Systems.ipynb

```diff
@@ -306,7 +306,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dowload large file from Sprinboard's shared GDrive folder\n",
```

Comment:
Could you take out the word Springboard here and just say "the shared GDrive"? That way it is transferable for UP versions.